pkg/kubelet/server/server.go

```diff
@@ -245,6 +247,7 @@ func NewServer(
 	} else {
 		server.InstallDebuggingDisabledHandlers()
 	}
+	server.InstallSystemLogHandler(enableSystemLogHandler)
```

Comment:
by decoupling this from the debugging handler option, a kubelet which previously set `--enable-debugging-handlers=false` in order to avoid serving logs would now serve logs.

Reply:
this is what I would expect to keep compatibility with existing behavior:
* if no debugging handler or log handler option is specified, serve logs
* if `--enable-debugging-handlers=false` and no log handler option is specified, do not serve logs
* if `--enable-debugging-handlers=true` and no log handler option is specified, serve logs

Reply:
Thanks @liggitt for the review. Sorry I missed this one, updated the PR to reflect all three scenarios. Basically copied the enableDebuggingHandler value to enableSystemLogHandler if enableSystemLogHandler is not specified.